cypress/e2e/tests/ci.test.ts

```
@@ -68,123 +63,232 @@ describe(["@ci"], "UI Sanity Tests", () => {
+        true,
+        CI_ALERT_TIMEOUT
+      );
+      notExists(archetype.name);
+    });
 
-    deleteByList(stakeholders);
-    deleteByList(stakeholderGroups);
-    deleteByList(tags);
+    // Clean up prerequisite entities via API
+    getAuthHeaders().then((headers) => {
+      tags.forEach((t) => t.deleteViaApi(headers));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)